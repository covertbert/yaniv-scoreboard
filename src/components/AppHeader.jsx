import React from 'react'
import styled from 'styled-components'
import { Link } from 'react-router-dom'
import { Icon } from 'gestalt'

const AppHeaderWrapper = styled.header`
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  padding: 20px;
  display: flex;
  justify-content: space-between;
`

const AppHeader = () => (
  <AppHeaderWrapper>
    <Link
      to="/"
      data-test-id="AppHeader/HomeButton"
    >
      <Icon
        accessibilityLabel="Home"
        icon="view-type-dense"
        size="25px"
      />
    </Link>

    <Link
      to="/player-management"
      data-test-id="AppHeader/PlayerManagementButton"
    >
      <Icon
        accessibilityLabel="Player Management"
        icon="people"
        size="25px"
      />
    </Link>
  </AppHeaderWrapper>
)

export default AppHeader
