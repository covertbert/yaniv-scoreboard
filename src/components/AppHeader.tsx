import { Icon } from 'gestalt'
import * as React from 'react'
import { Link } from 'react-router-dom'
import styledComponents from 'styled-components'

const AppHeaderWrapper = styledComponents.header`
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  padding: 20px;
  display: flex;
  justify-content: space-between;
`

const AppHeader = () => (
  <AppHeaderWrapper>
    <Link
      to="/"
      data-test-id="AppHeader/HomeButton"
    >
      <Icon
        accessibilityLabel="Home"
        icon="menu"
        size="25px"
      />
    </Link>

    <Link
      to="/player-management"
      data-test-id="AppHeader/PlayerManagementButton"
    >
      <Icon
        accessibilityLabel="Player Management"
        icon="people"
        size="25px"
      />
    </Link>
  </AppHeaderWrapper>
)

export default AppHeader
