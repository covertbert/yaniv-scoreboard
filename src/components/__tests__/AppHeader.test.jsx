import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import AppHeader from '../AppHeader'

describe('AppHeader', () => {
  it('renders the AppHeader', () => {
    const { wrapper } = shallow(<AppHeader />)
    console.log(wrapper.debug())


    // expect(wrapper).toContain('AppHeader/PlayerManagementButton')
    // expect(wrapper).toContain('AppHeader/HomeButton')
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
