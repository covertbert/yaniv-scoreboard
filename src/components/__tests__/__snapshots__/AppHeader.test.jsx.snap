// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader renders the AppHeader 1`] = `undefined`;
